chains/evm/contracts/pools/TokenPool.sol

```diff
@@ -59,10 +59,10 @@ abstract contract TokenPool is IPoolV1, Ownable2StepMsgSender {
   error OverflowDetected(uint8 remoteDecimals, uint8 localDecimals, uint256 remoteAmount);
   error InvalidDecimalArgs(uint8 expected, uint8 actual);
 
-  event Locked(address indexed sender, uint256 amount);
-  event Burned(address indexed sender, uint256 amount);
-  event Released(address indexed sender, address indexed recipient, uint256 amount);
-  event Minted(address indexed sender, address indexed recipient, uint256 amount);
+  event LockedOrBurned(uint64 indexed remoteChainSelector, address token, address sender, uint256 amount);
```

Comment:
Nit: can we have `token` before `remoteChainSelector`, for consistency with `Out|InboundRateLimitConsumed`), even if you need to `index` it?